docs/en/stack/ml/anomaly-detection/geographic-anomalies.asciidoc

```diff
@@ -0,0 +1,243 @@
+    },
+    "indices": [
+      "kibana_sample_data_ecommerce"
+    ]
+}
+
+POST _ml/anomaly_detectors/ecommerce-geo/_open <3>
+
+POST _ml/datafeeds/datafeed-ecommerce-geo/_start <4>
+{
+  "end": "2021-06-19T24:00:00Z"
+}
+--------------------------------------------------
+<1> Create the {anomaly-job}.
+<2> Create the {dfeed}.
+<3> Open the job.
+<4> Start the {dfeed}. Since the sample data sets often contain timestamps that
+are later than the current date, it is a good idea to specify the appropriate
+end date for the {dfeed}.
+====
+
+Alternatively, create a job that analyzes the sample web logs data set to detect 
+events with unusual coordinates (`geo.coordinates` values) or unusually high
+sums of transferred data (`bytes` values):
+
+[role="screenshot"]
+image::images/weblogs-advanced-wizard-geopoint.jpg[A screenshot of creating an {anomaly-job} using the web logs data in {kib}]
+
+.API example
+[%collapsible]
+====
+[source,console]
+--------------------------------------------------
+PUT _ml/anomaly_detectors/weblogs-geo <1>
+{
+  "analysis_config" : {
+    "bucket_span":"15m",
+    "detectors": [
+      {
+        "detector_description": "Unusual coordinates",
+        "function": "lat_long",
+        "field_name": "geo.coordinates"
+      },
+      {
+        "function": "high_sum",
+        "field_name": "bytes"
+      }
+    ]
+    "influencers": [
+      "geo.src",
+      "extension.keyword",
+      "geo.dest"
+    ]
+  },
+  "data_description" : {
+    "time_field": "timestamp",
+     "time_format": "epoch_ms"
+  }
+}
+
+PUT _ml/datafeeds/datafeed-weblogs-geo <2>
+{
+    "job_id": "weblogs-geo",
+    "query": {
+      "bool": {
+        "must": [
+          {
+            "match_all": {}
+          }
+        ]
+      }
+    },
+    "indices": [
+      "kibana_sample_data_logs"
+    ]
+}
+
+POST _ml/anomaly_detectors/weblogs-geo/_open <3>
+
+POST _ml/datafeeds/datafeed-weblogs-geo/_start <4>
+{
+  "end": "2021-07-15T22:00:00Z"
+}
+--------------------------------------------------
+<1> Create the {anomaly-job}.
+<2> Create the {dfeed}.
+<3> Open the job.
+<4> Start the {dfeed}. Since the sample data sets often contain timestamps that
+are later than the current date, it is a good idea to specify the appropriate
+end date for the {dfeed}.
+====
+
+[discrete]
+[[geographic-anomalies-results]]
+== Analyze the results
+
+After the {anomaly-jobs} have processed some data, you can view the results in
+{kib}.
+
+TIP: If you used APIs to create the jobs and {dfeeds}, you cannot see them
+in {kib} until you follow the prompts to synchronize the necessary saved objects.
+
+When you select a period that contains an anomaly in the swim lane results, you
+can see a map of the typical and actual coordinates. For example, the `jackson`
+user ID typically shops in Los Angeles so their purchase in New York is
+anomalous in the eCommerce sample data:
+//TBD: Is it working as designed that the map only appears after you click the swim lane?
```

Comment:
Should this be removed?
Also, the answer is yes (for data with geo point data) - it works as a type of chart - like the ones that appear already above the anomaly table once you click a swimlane.

Reply:
Thanks!  I've removed that comment.